state/store.go

```diff
@@ -9,6 +9,14 @@ import (
 	"github.com/tendermint/tendermint/types"
 )
 
+const (
+	// persist validators every valSetStoreInterval blocks to avoid
+	// LoadValidators taking too much time.
+	// https://github.com/tendermint/tendermint/pull/3438
+	// 100000 results in ~ 100ms to get 100 validators (see BenchmarkLoadValidators)
+	valSetStoreInterval = 100000
```

Comment:
Maybe,`valSetStoreBlockInterval` or `storeValSetHeightInterval` (it makes clear that every soandso blocks/height we store)